vtysh/vtysh_config.c

```diff
@@ -507,23 +540,40 @@ void vtysh_config_dump(void)
 				for (ALL_LIST_ELEMENTS(config->line, mnode,
 						       mnnode, line))
 					vty_out(vty, "%s\n", line);
+
+				configvec_dump(config->nested, true);
+
+				if (config->exit)
+					vty_out(vty, "%s\n", config->exit);
+
 				if (!NO_DELIMITER(i))
 					vty_out(vty, "!\n");
 
 				config_del(config);
 			}
-			if (NO_DELIMITER(i))
-				vty_out(vty, "!\n");
-		}
-
-	for (i = 0; i < vector_active(configvec); i++)
-		if ((configuration = vector_slot(configvec, i)) != NULL) {
 			config_master_fini(&configuration->master);
 			config_master_hash_fini(&configuration->hash_master);
 			XFREE(MTYPE_VTYSH_CONFIG, configuration);
-			vector_slot(configvec, i) = NULL;
+			vector_slot(vec, i) = NULL;
+			if (!nested && NO_DELIMITER(i))
+				vty_out(vty, "!\n");
 		}
+}
+
+/* Display configuration to file pointer. */
```

Comment:
Not seeing the file pointer in this function

Reply:
It's just an old comment that I didn't touch.
It's just git treating this as a change.
I will remove it.